src/main/java/com/minecolonies/coremod/colony/CitizenData.java

```diff
@@ -779,6 +779,21 @@ public void serializeViewNetworkData(@NotNull final ByteBuf buf)
 
         ByteBufUtils.writeUTF8String(buf, (job != null) ? job.getName() : "");
 
+        writeStatusToBuffer(buf);
+
+        buf.writeInt(colony.getID());
+
+        final NBTTagCompound compound = new NBTTagCompound();
+        compound.setTag("inventory", inventory.writeToNBT(new NBTTagList()));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use already-defined constant 'TAG_INVENTORY' instead of duplicating its value here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1192)